Dockerfile

```diff
@@ -1,5 +1,6 @@
 # syntax=docker/dockerfile:experimental
-FROM ubuntu:20.04 as baseline
+FROM ubuntu:22.04 as baseline
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Please clarify the need.

Reply:
Checking if still needed.

Reply:
Built an image and used it to perform a deployment, this is no longer needed. Removing.